packages/community-cli-plugin/src/commands/bundle/assetCatalogIOS.js

```diff
@@ -16,11 +16,11 @@ import fs from 'fs';
 import path from 'path';
 
 export function cleanAssetCatalog(catalogDir: string): void {
-  const files = fs
+  const imageSets = fs
     .readdirSync(catalogDir)
-    .filter(file => file.endsWith('.imageset'));
-  for (const file of files) {
-    fs.rmSync(path.join(catalogDir, file));
+    .filter(imageSet => imageSet.endsWith('.imageset'));
```

Comment:
nit: It's inaccurate to rename this to `imageSet` within this filter call, given these are candidates within the search directory. Let's use `file` or `fileOrDir`.

(I also don't mind the previous generic naming of `files` for the outer variable — until there is a future need to remove multiple kinds of files differently.)